### PR TITLE
Modifying getting started tutorial to match GE 101 webinar content

### PR DESCRIPTION
**Pull request by spbail**
As previously discussed, this is an updated version of the Getting started tutorial to match the GE webinar content more closely. Probably best to pull the branch and build docs locally to go through this.

**Major changes:**
- Changed out the data example (with a reference to the Docker container - see my open PR in the linked repo) and added a paragraph to lay out the data problem (TODO: add actual histograms of the January vs February data? Or set up the story a little differently?)
- Cut out quite a bit of "explanation" type content that was not critical path for the tutorial, for the sake of brevity
- Used scaffold instead of the init profiler to create the expectation suite - it actually generates the expectation we want (well, it checks for a value set rather than a min/max numeric range, but that works too!) - this is a change from the webinar script where we just run `suite new --empty` that I discussed with @abegong. I really like it since it allows us to show off scaffolding but also generate useful explanations for this particular exercise.
- Generally some more references to the actual data example, to make the exercise more tangible

### CODE REVIEW

File: docs/guides/tutorials/getting_started.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Getting started with Great Expectations
 ###############
 
-Welcome to Great Expectations! This tutorial will help you set up your first local deployment of Great Expectactions. We'll also introduce important concepts, with links to detailed material you can dig into later.
+Welcome to Great Expectations! This tutorial will help you set up your first local deployment of Great Expectations that contains a small Expectation Suite to validate some sample data. We'll also introduce important concepts, with links to detailed material you can dig into later.
 
-Please follow these steps to get started:
+The tutorial will walk you throw the following steps - **scroll to the bottom of this page and click the "Next" button to jump to the start of the tutorial!**
```
Comment (spbail) [inline docs/guides/tutorials/getting_started.rst:9]: I removed the individual links from each bullet here and just kept the explanations as an overview of the tutorial content, since I found it really confusing to have links here, instead of just following the "next" button.

File: docs/guides/tutorials/getting_started/connect_to_data.rst [hunk 2]
```diff
@@ -14,88 +14,71 @@ Let's configure your first Datasource, by following the next steps in the CLI in
...
 
-When you completed those last few steps in ``great_expectations init``, you told Great Expectations that
+When you completed those last few steps in ``great_expectations init``, you told Great Expectations that:
 
-1. You want to create a new Datasource called ``my_data__dir``.
-2. You want to use Pandas as your :ref:`Execution Engine <Execution Engines>`, hence ``data_asset_type.class_name = PandasDataset``.
-3. You want to create a BatchKwarg Generator called ``subdir_reader`` using the class ``SubdirReaderBatchKwargsGenerator``.
-4. This particular Generator connects to data in files within a local directory, specified here as ``../my_data``.
+1. You want to create a new Datasource called ``my_postgres_db``.
+2. You want to use Postgres as your :ref:`Execution Engine <Execution Engines>`, hence ``data_asset_type.class_name = SqlAlchemyDataset``.
```
Comment (spbail) [inline docs/guides/tutorials/getting_started/connect_to_data.rst]: I can't actually find a single reference to Execution Engine in the docs. I left the ref as a placeholder - which page should this link to?

File: docs/guides/tutorials/getting_started/validate_your_data.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
+.. _tutorials__getting_started__validate_your_data:
```
Comment (spbail) [inline docs/guides/tutorials/getting_started/validate_your_data.rst:1]: The content is still mostly the old "create a checkpoint" page. I renamed it since the tutorial audience may not know what checkpoint means, so "validation" seemed clearer.

**Review by spbail — COMMENTED** [review]
Added a couple of comments to explain changes/reasoning behind it, but even just looking at this PR on github, it shouldn't be too hard to see what to review.

**Comment by spbail** [comment]
CC @eugmandel

File: docs/guides/how_to_guides/creating_and_editing_expectations/spare_parts/how_to_create_expectations.rst [hunk 4]
```diff
@@ -1,4 +1,4 @@
-.. _tutorial_create_expectations:
+.. _tutorials__create_expectations:
 
 How to create Expectations
 ==========================
```
Comment (jcampbell) [inline docs/guides/how_to_guides/creating_and_editing_expectations/spare_parts/how_to_create_expectations.rst:4]: Warning below can now be removed!

Comment (jcampbell) [reply docs/guides/how_to_guides/creating_and_editing_expectations/spare_parts/how_to_create_expectations.rst:4]: Also, please capitalize "Expectations" per new style guide in the steps to be looped over (again, realize this isn't from you originally, but while we're editing this tutorial).

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 5]
```diff
@@ -1,9 +1,9 @@
 .. _tutorials__getting_started__initialize_a_data_context:
 
-Initialize a Data Context
+Set up the tutorial data and initialize a Data Context
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst:3]: Please make analogous change in the name of the step in the Getting Started document to maintain parallelism. (currently, it still reads "Initialize a DataContext")

(small nit as well: Sphinx produces a warning if the length of the line of header-delimiters (e.g. '=' in this case) is less than the header itself).

File: docs/guides/tutorials/getting_started.rst (hunk 1 above)
Comment (jcampbell) [reply docs/guides/tutorials/getting_started.rst:9]: I agree. In fact, I think we should remove the huge numbers and font and make the overview here much slimmer, since you're going to recommend scrolling past it.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 ===============================================
 
-In Great Expectations, your :ref:`Data Context` manages boilerplate configuration. Using a Data Context is almost always the fastest way to get up and running, even though some teams don't need every component of a Data Context.
+In Great Expectations, your :ref:`Data Context` manages your project configuration. Using a Data Context is almost always the fastest way to get up and running, even though some teams don't need every component of a Data Context.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst:6]: I strongly prefer the new "your project" wording, and would make the same change in the overview description of this step in the getting started (overview) doc.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 7]
```diff
@@ -24,64 +24,59 @@ or git, you may want to check out the :ref:`supporting_resources` section before
...
 
-If you intend to develop within the Great Expectations (e.g. to contribute back to the project), check out :ref:`contributing_setting_up_your_dev_environment` in the contributor documentation.
+For this tutorial, we will use a simplified version of the NYC taxi ride data. We prepared a Docker image that contains a local Postgres database with the data pre-loaded, so you can easily get up and running without any local dependencies.
 
-Download example data
----------------------
+To avoid confusion during the tutorial, we recommend you follow these steps:
 
-For this tutorial, we will use a simplified version of the National Provider Identifier (NPI) database. It's a public dataset released by the `Centers of Medicare and Medicaid Services <https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/NationalProvIdentStand/DataDissemination>`_, intended as an authoritative list of health care providers in the United States. NPI data is famously messy---a great place to see the value of data testing and documentation in action.
+#. Make sure you have `Docker<https://www.docker.com/>`_ installed
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: Broken link (probably just needs a space before the '<')

image

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 8]
```diff
@@ -24,64 +24,59 @@ or git, you may want to check out the :ref:`supporting_resources` section before
...
 
-To avoid confusion during the tutorial, we recommend you set up the following directory structure before you download the data:
+#. Clone the ``ge_tutorials`` repository and start up the container with the Postgres database containing the data:
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: Turn ge_tutorials into an external link?

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 9]
```diff
@@ -24,64 +24,59 @@ or git, you may want to check out the :ref:`supporting_resources` section before
...
-   mkdir example_project/my_data
-   cd example_project
+   git clone https://github.com/superconductive/ge_tutorials
+   cd ge_tutorials/ge_getting_started_tutorial
+   docker-compose up
 
-To download the NPI data using wget, please run:
-
-.. code-block:: bash
+You will now have a Postgres database running with some pre-loaded data! In case you're looking to connect to the database, you'll find instructions in the ``README`` in the repository.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst]:
```suggestion
You will now have a Postgres database running with some pre-loaded data! In case you're looking to connect to the database, you'll find instructions in the `README <https://github.com/superconductive/ge_tutorials>`_ in the repository.
```

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 10]
```diff
@@ -24,64 +24,59 @@ or git, you may want to check out the :ref:`supporting_resources` section before
...
 
-    wget https://superconductive-public.s3.amazonaws.com/data/npi/weekly/npidata_pfile_20200511-20200517.csv.gz -P my_data
+About the data
+-----------------------------------------------
 
-Alternatively, you can use curl:
+The NYC taxi data is an open data set which is updated every month. Each record corresponds to one taxi ride and contains information such as the pick up and drop-off location, the payment amount, and the number of passengers, among others.
 
-.. code-block:: bash
+In this tutorial, we provide two tables with a 10,000 row sample each of the taxi data:
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst]:
```suggestion
In this tutorial, we provide two tables, each with a 10,000 row sample of the taxi data:
```

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 11]
```diff
@@ -24,64 +24,59 @@ or git, you may want to check out the :ref:`supporting_resources` section before
...
+* **yellow_tripdata_staging**: a sample of the February 2019 taxi data, loaded to a "staging" table so we can validate it before promoting it to a permanent table
 
-Finally, to unzip the data, please run:
+If we compare the ``passenger_count`` column in the January and February data, we find a significant difference: The February data contains a large proportion of rides with 0 passengers, which seems unexpected.
 
-.. code-block:: bash
+.. admonition:: The data problem we're solving in this tutorial
 
-    gunzip my_data/npidata_pfile_20200511-20200517.csv.gz
+    In this tutorial, we will be creating an Expectation Suite for this example data set that allows us to assert that we generally expect at least 1 passenger per taxi ride based on what we see in the January 2019 data. We will then use that Expectation Suite to catch the data quality issue in the February 2019 staging data.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst:63]: <3 <3 <3

So clear and to the point.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst [hunk 12]
```diff
@@ -117,7 +112,10 @@ You should see this:
 
     OK to proceed? [Y/n]: 
 
-Let's pause there for a moment.
+**Let's pause there for a moment and take a look under the hood.**
+
+The ``great_expectations/`` directory structure
+-----------------------------------------------
 
 Once you finish going through ``init``, your ``great_expectations/`` directory will contains all of the important components of a Great Expectations deployment, in miniature:
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/initialize_a_data_context.rst]:
```suggestion
Once you finish going through ``init``, your ``great_expectations/`` directory will contain all of the important components of a Great Expectations deployment:
```

File: docs/guides/tutorials/getting_started/connect_to_data.rst [hunk 13]
```diff
@@ -14,88 +14,71 @@ Let's configure your first Datasource, by following the next steps in the CLI in
...
     
-    Would you like to proceed? [Y/n]: 
-
-That's it! You just configured your first Datasource!
-
-Before continuing, let's stop and unpack what just happened, and why.
-
-Why Datasources?
-----------------
+    Next, we will configure database credentials and store them in the `my_postgres_db` section of this config file: great_expectations/uncommitted/config_variables.yml:
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/connect_to_data.rst]: Maybe add a manual linebreak here? In my build-docs view, this scrolls off screen instead of wrapping.

File: docs/guides/tutorials/getting_started/connect_to_data.rst (hunk 2 above)
Comment (jcampbell) [reply docs/guides/tutorials/getting_started/connect_to_data.rst]: I think you're right -- it's not there.

I think I'd reword this as follows:
```suggestion
2. You want to use SqlAlchemy to evaluate your Expectations, hence ``data_asset_type.class_name = SqlAlchemyDataset``. Your database configuration tells the SqlAlchemyDataset to use a Postgresql-specific driver.
```
The term execution engine *does* appear in the draft updated core concepts doc, but it's better to let all those names settle before surprising users with this one out of the blue.

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst [hunk 14]
```diff
@@ -3,44 +3,67 @@
...
-    
-    Generating example Expectation Suite...
-    
-    Done generating example Expectation Suite
+
+
+Which data should I choose when creating Expectations?
+====================
+
+In this example, we want to build an Expectation Suite based on what we know about our taxi data: each taxi ride has a ``passenger_count`` between 1 and 6. We then want to use that Expectation Suite to validate any future data that is loaded into the ``staging`` table. Hence, we choose the ``yellow_tripdata_sample_2019_01`` table when creating the new Expectation Suite, which we will then use to validate the ``yellow_tripdata_staging`` table **in a later step**. Makes sense, right?
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/create_your_first_expectations.rst:36]: Again, I *love* the earlier introduction of batches. It feels *so* much more real.

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst [hunk 15]
```diff
@@ -3,44 +3,67 @@
...
+
+After selecting the table, Great Expectations will open a Jupyter notebook which will take you through the next part of the ``scaffold`` workflow.
+
+
+Creating Expectations in Jupyter notebooks
+-------------------
+
+In this section, we will explain how to create and edit Expectations in a Jupyter notebook. Notebooks are a simple way of interacting with the Great Expectations Python API. You could also just write all this in plain Python code, but for convenience, Great Expectations provides you some boilerplate code in notebooks.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/create_your_first_expectations.rst:44]: I strongly prefer the sentence here "Notebooks are a simple way of interacting with the Great Expectations Python API" to the description of them as the "best way" to do anything....and I don't see that in this PR, so just a FYI. :)

Comment (jcampbell) [reply docs/guides/tutorials/getting_started/create_your_first_expectations.rst:44]:
```suggestion
In this section, we will explain how to create and edit Expectations in a Jupyter notebook. Notebooks are a simple way of interacting with the Great Expectations Python API. You could also just write all this in plain Python code, but for convenience, Great Expectations provides you some boilerplate code in notebooks.
Since notebooks are often less durable, creating Expectations in a notebook also helps reinforce that the source of truth about Expectations is the Expectation Suite, **not** the code that generates the Expectations.
```

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst [hunk 16]
```diff
@@ -3,44 +3,67 @@
...
+
+    included_columns = [
+        # 'vendor_id',
+        # 'pickup_datetime',
+        # 'dropoff_datetime',
+        'passenger_count',
+        ...
+    ]
+
+#. The next cell passes the scaffolder config to the ``BasicSuiteBuilderProfiler``, which will then profile the data and create the relevant Expectations to add to your ``taxi.demo`` suite.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/create_your_first_expectations.rst]: I think "profiler" config is better here, since I don't think "scaffolder" is really precisely used (i.e. it's just a way of using a profiler or even intent for using a profiler)

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst [hunk 17]
```diff
@@ -3,44 +3,67 @@
...
+
+After selecting the table, Great Expectations will open a Jupyter notebook which will take you through the next part of the ``scaffold`` workflow.
+
+
+Creating Expectations in Jupyter notebooks
+-------------------
+
+In this section, we will explain how to create and edit Expectations in a Jupyter notebook. Notebooks are a simple way of interacting with the Great Expectations Python API. You could also just write all this in plain Python code, but for convenience, Great Expectations provides you some boilerplate code in notebooks.
+
+Here's what's happening in the notebook:
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/create_your_first_expectations.rst:51]: I found it a little jarring not to have the notebook on screen at this point for myself if I am just reading the tutorial instead of following along. If there's a way to embed it, I think that might be nice. If not, I think this works fine (it wasn't a major jolt).

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst [hunk 18]
```diff
@@ -84,39 +108,33 @@ If you open up the suite in ``great_expectations/expectations/npidata_pfile_2020
...
-              "Provider Name Prefix Text",
-              "Provider Name Suffix Text",
-              "Provider Credential Text",
         ...
 
-There's a lot of information here. (This is good.)
+There's a lot of information in the JSON file. **We will focus on just the snippet above:**
 
-Every Expectation in the file expresses a test that can be validated against data. (This is very good.)
+Every Expectation in the file expresses a test that can be validated against data. You can see that the Profiler generated several Expectations based on our data, including ``expect_column_distinct_values_to_be_in_set``, with the ``value_set`` containing the numbers 1 through 6. This is exactly what we wanted: An assertion that the ``passenger_count`` column contains those values!
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/create_your_first_expectations.rst]: 👍

File: docs/guides/tutorials/getting_started/set_up_data_docs.rst [hunk 19]
```diff
@@ -3,65 +3,23 @@
...
-     - local_site: file:///home/ubuntu/example_project/great_expectations/uncommitted/data_docs/local_site/index.html
-    
-    Would you like to proceed? [Y/n]: 
-    
-    Building Data Docs...
-    
-    Done building Data Docs
-    
-    Would you like to view your new Expectations in Data Docs? This will open a new browser window. [Y/n]: 
+The validation results page is *descriptive*: it describes how a specific batch of data (in this case, the data we profiled) actually looked when validated. The validation results page shows you both your *Expectation* in human-readable form, as well as the *Observed Value* that was found in the data batch you validated, and a *Status* that tells you whether the observed value *passed* or *failed* the validation.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/set_up_data_docs.rst]: I want to clarify that Validation Results pages are both prescriptive (like an expectation page) **and** descriptive (which we also display for cases where there are no expectations or only vacuous expectations). The descriptive-only renderer is not covered in this tutorial.


```suggestion
The validation results page is *descriptive*: it describes how a specific batch of data (in this case, the data we profiled) actually looked when validated, by showing the *Observed Value* that was found in the data batch. The validation results page is **also** *prescriptive*: it shows you your normative *Expectation* in human-readable form. Comparing what was expected and what was observed lets the Validation Result provide a *Status* that tells you whether the observed value *passed* or *failed* the validation.
```

File: docs/guides/tutorials/getting_started/set_up_data_docs.rst [hunk 20]
```diff
@@ -3,65 +3,23 @@
...
-
-.. <<<SCREENSHOT GOES HERE>>>
-
-.. Finally,
-
-In addition, your static site includes a set of pages for future data validations. These pages are *diagnostic*: they describe whether and how a specific batch of data differed from the ideal prescribed by a set of Expectations.
-
-.. figure:: ../../images/diagnostic_view.png
+#. If you click on the *Home* page, you will see a list of all validation runs.
+#. The *Home* page also has a tab for your Expectation Suites. The Expectation Suite pages show you the *prescriptive* view of your data.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/set_up_data_docs.rst]:
```suggestion
#. The *Home* page also has a tab for your Expectation Suites. The Expectation Suite pages show you the exclusively *prescriptive* view of your data.
```

File: docs/guides/tutorials/getting_started/validate_your_data.rst (hunk 3 above)
Comment (jcampbell) [reply docs/guides/tutorials/getting_started/validate_your_data.rst:1]: +1

File: docs/guides/tutorials/getting_started/validate_your_data.rst [hunk 21]
```diff
@@ -0,0 +1,97 @@
...
+This will output the following:
+
+.. code-block:: bash
+
+    Heads up! This feature is Experimental. It may change. Please give us your feedback!
+    Validation Failed!
+
+**What just happened?**
+
+We ran the Checkpoint and it successfully failed! **Wait - what?** Yes, that's correct, and that's we wanted. We know that in this example, the staging data has data quality issues, which means we *expect* the validation to fail. Let's open up Data Docs again to see the details.
```
Comment (jcampbell) [inline docs/guides/tutorials/getting_started/validate_your_data.rst:74]: <3 style

**Review by jcampbell — CHANGES_REQUESTED** [review]
This is amazing. I made quite a few small comments, but they should all be quickly addressable. LMK of any questions!

One add-on:
Could you please update this (I found it inadvertently when I was going through the as-built docs for the review)

image

File: docs/guides/tutorials/getting_started.rst (hunk 1 above)
Comment (abegong) [reply docs/guides/tutorials/getting_started.rst:9]: Removing the links makes sense to me. I still like the visual hierarchy of the numbers as a way to make the flow clear.

File: docs/guides/tutorials/getting_started/connect_to_data.rst (hunk 2 above)
Comment (abegong) [reply docs/guides/tutorials/getting_started/connect_to_data.rst]: :+1:

File: docs/guides/tutorials/getting_started/validate_your_data.rst (hunk 3 above)
Comment (abegong) [reply docs/guides/tutorials/getting_started/validate_your_data.rst:1]: +1

**Review by abegong — APPROVED** [review]
I already made a couple notes in comments. (Sry!)

Overall, this is awesome, awesome, awesome. Great narrative; tight, intuitive example; great attention to flow.

I don't love that we need to drop out of the `init` flow halfway through in order to scaffold, but I agree that this makes for a better tutorial. IMO, we should pick our moment, then change `init` to match.

Also, I don't love that both profiling and checkpoints are flagged as "experimental." ...And they're both experimental, so we just have to live with it for now.

I agree with all of @jcampbell 's comments except for one. IMO, we should reserve the word "descriptive" for the case that is strictly describing data, usually as part of a profiler.

I'd propose something along these lines:
	* Descriptive : describes what the data looked like, with no direct reference to Expectations (Common for profiler output.) “The column mean is 30”
	* Prescriptive : describes Expectations directly. “The column mean should be between 0 and 20.”
	* Diagnostic (?) : describes whether or not a given data set met Expectations, possibly including metadata to understand failed validation. “The column mean was 30, but it should have been between 0 and 20.”

**Comment by jcampbell** [comment]
I think we are really close on this. Here's the picture in my head:

image

I think "Validation Results Page" is built with a "Diagnostic" renderer. I just think it's correct to say that what it's doing is using the joint prescriptive and descriptive information.

Changing the wording to drop *both* prescriptive and descriptive and *only* say diagnostic (in the key para) also works for me--that might be clearer (though I do like making the point that it's really about the two together). Let's see what Sam's preference is.

**Comment by abegong** [comment]
Aligned. That picture+explanation should go in our References section somewhere.

**Comment by jcampbell** [comment]
...it's on the current draft-one-pager of core concepts (pending the design updates/icons/graphic design...I made that with circles and text and I think Kyle was....underwhelmed....at my visual prowess).

**Comment by abegong** [comment]
It looks like the Airbnb logo. What's not to like?

File: docs/guides/how_to_guides/creating_and_editing_expectations/spare_parts/how_to_create_expectations.rst (hunk 4 above)
Comment (spbail) [reply docs/guides/how_to_guides/creating_and_editing_expectations/spare_parts/how_to_create_expectations.rst:4]: Yup, cleaned up.

File: docs/guides/tutorials/getting_started.rst (hunk 1 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started.rst:9]: I'll leave it as-is for now, but I think it's a lot of reading just for a table of contents.

File: docs/guides/tutorials/getting_started/connect_to_data.rst (hunk 13 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/connect_to_data.rst]: Good catch, done.

File: docs/guides/tutorials/getting_started/connect_to_data.rst (hunk 2 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/connect_to_data.rst]: Cool, thanks!

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst (hunk 15 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/create_your_first_expectations.rst:44]: Added your paragraph, but made it "less permanent" instead of "less durable", just to simplify the language a little?

**Comment by jcampbell** [comment]
@spbail  -> #1842 flags several broken links that I think are flagged in this PR as well. Can you confirm for when this PR merges.

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst (hunk 17 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/create_your_first_expectations.rst:51]: I added an animated gif of the notebook, I really like it actually!

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst (hunk 16 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/create_your_first_expectations.rst]: Good call.

File: docs/guides/tutorials/getting_started/create_your_first_expectations.rst (hunk 18 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/create_your_first_expectations.rst]: I mean, it's using a value set instead of a numeric range, but I think that's cool, too.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst (hunk 6 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/initialize_a_data_context.rst:6]: Good catch, I had changed that in the first iteration of this PR but then didn't go back. And yes, I'm trying to simplify/clarify the language a little.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst (hunk 7 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: The rst struggle is real.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst (hunk 8 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: Good idea!

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst (hunk 9 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: Changed the link to point directly to the right subdirectory.

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst (hunk 10 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: 👍

File: docs/guides/tutorials/getting_started/initialize_a_data_context.rst (hunk 12 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/initialize_a_data_context.rst]: Changed this and added "local" deployment, for consistency. As an aside, I'm making changes on my local branch, so not accepting them through github.

File: docs/guides/tutorials/getting_started/set_up_data_docs.rst (hunk 20 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/set_up_data_docs.rst]: Added

File: docs/guides/tutorials/getting_started/set_up_data_docs.rst (hunk 19 above)
Comment (spbail) [reply docs/guides/tutorials/getting_started/set_up_data_docs.rst]: Added

**Review by spbail — COMMENTED** [review]
Right back atcha. I think I've incorporated all comments and made just a couple of tiny additional tweaks for formatting.

**Comment by jcampbell** [comment]
I think my first comment from a few days ago slipped through the cracks (because it wasn't part of this PR's changes originally), but can you please update the broken links flagged in issue 1842, e.g. the following from the getting started page?

image

**Comment by spbail** [comment]
@jcampbell  I fixed the broken links in another PR #1846 - I think I should merge that one first, then rebase this one and hope for the best, yay.

**Review by jcampbell — APPROVED** [review]

**Comment by spbail** [comment]
Rebase wasn't too bad actually...
